editors/nvi/Portfile

```diff
@@ -18,7 +18,8 @@ platforms       darwin
 master_sites    ftp://ftp.stack.nl/pub/users/johans/nvi/
 checksums       md5 88d1e23115ee9f2961186b62e55f5704 \
                 rmd160 0db8568bea96392d9a027044177c60317c8ade36 \
-                sha256 3441b8e3247b2b6dd19b5fd38355fe6593b2a92e960ecb3dd00d676f63d06b59
+                sha256 3441b8e3247b2b6dd19b5fd38355fe6593b2a92e960ecb3dd00d676f63d06b59 \
+				size 1758309
```

Comment:
Wrong alignment:
```suggestion
		size 1758309
```

Reply:
I accidentally used tabs instead of spaces.  This should be fixed in my latest commit.